templates/master/01-master-kubelet/alibabacloud/units/kubelet.service.yaml

```diff
@@ -39,7 +39,7 @@ contents: |
         --cloud-provider={{cloudProvider .}} \
```

Comment:
Tangentially related, the forking of the kubelet unit is definitely out of control now :cry: